travis.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
```

Comment:
What happened to all of the whitespace?

Also, this file should be executable.